capstone/src/main/java/com/google/step/data/RestaurantScore.java

```diff
@@ -0,0 +1,46 @@
+package com.google.step.data;
+
+/**
+ * Class that holds the score for a specific restaurant.
+ */
+public class RestaurantScore {
```

Comment:
How is this class used? I'm wondering if a map from restaurant id to score might be easier to work with.

Reply:
It's used for the score client (which ill put in the next PR), where I return a list of this class with each restaurant and score. The map could also be the case but I prefer this method just so Josh doesn't need to know the Id's before-hand and can just have them in the same class in order to update his search indexes.

Reply:
If Josh is iterating over the whole list, he should be to use a loop like this with a map:
https://www.geeksforgeeks.org/iterate-map-java/

Reply:
So currently I changed everything to be final and I will be changing the client accordingly.